src/app/todo/components/todoAdding/todoAdding.component.ts

```diff
@@ -63,15 +62,39 @@ export class TodoAddingComponent implements ITask {
     const formattedDate = `${formattedDay}.${formattedMonth}.${formattedYear}`;
     const id = Date.now();
 
-    const task: ITask = {
-      id: id,
-      title: this.title,
-      description: this.description,
-      type: this.type,
-      createdOn: formattedDate,
-      status: this.status,
-    };
-    console.log(task);
-    this.taskService.addTask(task);
+    switch (this.taskForm.value.rawType) {
+      case 1:
+        this.type = this.TaskTypes.LowUrgency;
+        break;
+      case 2:
+        this.type = this.TaskTypes.MediumUrgency;
+        break;
+      case 3:
+        this.type = this.TaskTypes.HighUrgency;
+        break;
+    }
+    if (this.taskForm.value.title && this.taskForm.value.description) {
+      const task: ITask = {
+        id: id,
+        title: this.taskForm.value.title,
+        description: this.taskForm.value.description,
+        type: this.type,
+        date: formattedDate,
+        status: this.status,
+      };
+      // this.taskService.addTask(task);
+      this.taskApi.post('http://localhost:3001/api/todo', task).subscribe({
```

Comment:
you can define API and URLs for endpoints in a separate file like constants.ts and in environment file define the base url (which in this case would be http://localhost:3001/) instead of passing the variables in the component